takora.spec

```diff
@@ -0,0 +1,34 @@
+Name:           takora
+Version:        0.1.0
+Release:        1%{?dist}
+Summary:        Puppet modules for Ceph installation
+License:        ASL 2.0
+
+URL:            https://github.com/ceph/takora
+
+#Source0:        https://github.com/ceph/%{name}/archive/%{version}.tar.gz
+Source0:       %{name}-%{version}.tar.gz
+
+BuildArch:      noarch
+
+%description
+A collection of Puppet modules which are required to install and configure
+Ceph via installers using Puppet configuration tool.
+
+%prep
+%setup -q -n %{name}-%{version}
```

Comment:
"`-n %{name}-%{version}`" is the default, so you can just do "`%setup -q`" without the `-n` option